docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1195,358 +1195,810 @@ thor:
+            </listitem>
 
-      <para>The certificate system integrates with cert-manager to provide
-      automated certificate generation and renewal for both internal (local,
-      i.e., communication between HPCC components within the same cluster or
-      namespace) and external (public/remote) communications.</para>
+            <listitem>
+              <para>Set the component's <emphasis>visibility</emphasis>
+              property to the value that assigns it the issuer you want to
+              use: `cluster` visibility for `local` issuer, `local` or
+              `global` visibility for the `public` issuer.</para>
+            </listitem>
 
-      <sect3 id="Container_Certificates">
-        <title>Enable Certificates</title>
+            <listitem>
+              <para>To use the `remote` issuer, set the component's
+              `visibility` property to `local` or `global`, and its
+              `trustClients` array to the list of remote trusted
+              clients.</para>
+
+              <variablelist>
+                <varlistentry>
+                  <term>Note:</term>
+
+                  <listitem>
+                    <para>Although an effort was made to standardize on the
+                    term TLS throughout this section, there may be references
+                    to the older SSL standard it replaced. Any SSL references
+                    should be taken as colloquial or legacy nomenclature that
+                    actually refers to a TLS implementation.</para>
+                  </listitem>
+                </varlistentry>
+              </variablelist>
+            </listitem>
+          </itemizedlist></para>
 
-        <para>Use the certificates section to enable cert-manager to generate
-        TLS certificates for each component in the HPCC Systems
-        deployment.</para>
+        <para>The following examples are to exhibit basic functionality in a
+        testing environment- production deployments may require different
+        settings. The instructions reference paths that are relative to the
+        root of the HPCC Systems Platform repository. Adjust as needed for
+        your environment.</para>
 
-        <programlisting>certificates:
-  enabled: false
-  issuers:
-    local:
-      name: hpcc-local-issuer</programlisting>
+        <para>A prerequisite to all these examples is installing cert-manager
+        to your Kubernetes cluster:</para>
 
-        <para>In the delivered YAML file certificates are not enabled, as
-        illustrated above. You must first install the cert-manager to use this
-        feature.</para>
-      </sect3>
+        <para>Add Jetstack helm repo: <programlisting>helm repo add jetstack https://charts.jetstack.io</programlisting>
+        Install cert-manager. <programlisting>helm install cert-manager jetstack/cert-manager --set crds.enabled=true --namespace cert-manager --create-namespace</programlisting></para>
```

Comment:
Perhaps make this a sect4 "Prerequisite" element to make it stand out, and simplify and correct the instructions like: 
```
<para>Install cert-manager: <programlisting>helm repo add jetstack https://charts.jetstack.io
helm repo update
helm install cert-manager jetstack/cert-manager --set crds.enabled=true --namespace cert-manager --create-namespace</programlisting></para>
```